src/Languages/Core/Impl/Text/CharacterStream.cs

```diff
@@ -207,15 +207,15 @@ public CharacterStream(string text)
         /// <summary>
         /// Determines if current character is an ANSI letter
```

Comment:
This comment, and the one below, are wrong now, since it checks for a Unicode letter.